Partition (WBP)/Functions/VI0049C.VTI0049.fun.yml

```diff
@@ -18,31 +18,42 @@
     
     Define Field(#LF_i) Reffld(#Std_Count)
     Define Field(#LF_Count) Reffld(#Std_Count)
-    Define Field(#LF_rc) Reffld(#BIFA002A)
-    Define Field(#LF_logfile_name) Reffld(#BIFA010A)
-    Define Field(#LF_commandLine_script) Reffld(#BIFA010A)
+    Define Field(#LF_rc) Reffld(#BIFA002A) Default(OK)
+    Define Field(#LF_logfile_name) Reffld(#BIFA256A)
+    Define Field(#LF_commandLine_script) Reffld(#BIFA256A)
+    
+    Define Field(#LF_osbit) Type(*DEC) Length(2) Decimals(0)
+    * DB Type
+    Define Field(#LF_DB_Type) Type(*CHAR) Length(32)
+    Use Builtin(Get_Session_Value) With_Args(DBUT) To_Get(#LF_DB_Type #LF_rc)
     
     * Start testing.
     Clr_List Named(#wLRet)
     #Com_Owner.mthAdd_Msg( OK *Function ('Start - ' + *Function_Desc) )
+    Use Builtin(CLR_MESSAGES)
+    
     Get_Entry Number(1) From_List(#wLKData)
     
+    * #LF_Execution_Type := W
+    * #LF_Local_Platform := W
+    Change Field(#LF_osbit) To(*OSBITNESS)
+    
+    #Com_Owner.mthGetExecutionType()
+    
     * DELETE / INSERT TEST DATA.
     #com_owner.mthDoDelete( #LF_rc )
     #Com_Owner.mthInsert( #LF_rc )
     
-    *
-    * Execute SQL script
-    *
-    * Initialize local variables.
-    * #LF_rc := OK
-    * #LF_logfile_name := 'c:\temp\PerformanceMetricsLog.txt'
-    * #LF_commandLine_script := 'SQLCMD -S <servername/InstanceName> -U -P -i "C:\Program Files (x86)\Lansa\LANSA\VersionControl\Scripts\mssql-performance-statistics-bind.sql'
-    *
-    * Use Builtin(SYSTEM_COMMAND) With_Args('X' #LF_commandLine_script ' ' #LF_logfile_name) To_Get(#STD_NUM)
-    * If Cond(#LF_rc = ER)
-    * #Com_Owner.mthAdd_SubMsg( ER ('- Log file path does not exist: ' + #LF_logfile_name) ('') )
-    * Endif
+    * Execute SQL Script
+    If ((#LF_Execution_Type *EQ W) *And (#LF_Local_Platform *EQ W))
```

Comment:
The whole test needs to be wrapped in this test. That is, including Delete and Insert